clouditor-engine-core/src/main/java/io/clouditor/assurance/Control.java

```diff
@@ -128,7 +156,7 @@ public boolean equals(Object o) {
     var control = (Control) o;
 
     return new EqualsBuilder()
-        .append(rules, control.rules)
+        .append(new ArrayList<>(rules), new ArrayList<>(control.rules))
```

Comment:
What's the point of that?

Reply:
It is to genearate equality between a org.hibernate.collection.internal.PersistentBag and a java.util.List